sdks/java/io/google-cloud-platform/expansion-service/build.gradle

```diff
@@ -34,6 +34,8 @@ dependencies {
     permitUnusedDeclared project(":sdks:java:expansion-service") // BEAM-11761
     implementation project(":sdks:java:io:google-cloud-platform")
     permitUnusedDeclared project(":sdks:java:io:google-cloud-platform") // BEAM-11761
+    implementation project(":sdks:java:io:kafka")
```

Comment:
gcp expansion service should not depend on kafka. Note that expansion service jar is a shadowJar. All transient dependencies will be added.

Consider make the test suite depend on project(`:sdks:java:io:expansion-service`) and invoke the io-expansion-service in kafka test